TddShop/TddShop.Cli/Shipment/AncientRomeShippingService.cs

```diff
@@ -20,7 +23,23 @@ public AncientRomeShippingService(IDeliveryService deliveryService)
         /// <param name="order"></param>
         public void ShipOrder(OrderModel order)
         {
-            
+            if (ValidOrder(order))
+            {
+                int shipmentRefNumberArabic = _deliveryService.GenerateShipmentReferenceNumber(order.Items.Length);
+                string shipmentRefNumberRoman = _romanConverter.Convert(shipmentRefNumberArabic);
+
+                _deliveryService.RequestDelivery(shipmentRefNumberRoman, order);
+            }
+        }
+
+        private bool ValidOrder(OrderModel order) {

```

Comment:
Fajnie, że wydzieliłeś metodę ValidOrder. Zwróć uwagę, że klasa Item ma pole Quantity. Tak jak na warsztatach - powinienneś dodać do siebie wszystkie Quantity.

Reply:
No właśnie tak myślałem że o czymś zapomniałem i po to ją wydzieliłem żeby łatwiej było jeszcze jakiś przypadek dodać/edytować.